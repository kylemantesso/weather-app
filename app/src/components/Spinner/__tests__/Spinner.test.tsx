import React from "react";
import renderer from "react-test-renderer";
import { Spinner } from "../Spinner";

describe("Spinner", () => {
  it("matches snapshot", () => {
    const component = renderer.create(<Spinner />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
