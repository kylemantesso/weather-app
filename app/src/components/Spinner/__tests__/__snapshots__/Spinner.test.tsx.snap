// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner matches snapshot 1`] = `
<div
  className="StyledBox-sc-13pk1d4-0 fHyGHh"
>
  <svg
    fill="#7D4CDB"
    height="28px"
    version="1.1"
    viewBox="0 0 32 32"
    width="28px"
  >
    <path
      d="M16 0 A16 16 0 0 0 16 32 A16 16 0 0 0 16 0 M16 4 A12 12 0 0 1 16 28 A12 12 0 0 1 16 4"
      opacity=".25"
    />
    <path
      d="M16 0 A16 16 0 0 1 32 16 L28 16 A12 12 0 0 0 16 4z"
    >
      <animateTransform
        attributeName="transform"
        dur="0.8s"
        from="0 16 16"
        repeatCount="indefinite"
        to="360 16 16"
        type="rotate"
      />
    </path>
  </svg>
</div>
`;
